### ISSUE TITLE
OPENVPN 

### ISSUE THREAD

**Issue by NYFDmods** [angristan/openvpn-install#93]
for some odd reason, when i enter commands, it states no files in directory. any ideas y

**Comment by angristan** [comment]
Please do the following : 
```
cd ~
wget https://raw.githubusercontent.com/Angristan/OpenVPN-install/master/openvpn-install.sh
chmod +x openvpn-install.sh
./openvpn-install.sh
```

And paste here what you get.

**Comment by angristan** [comment]
No answer

**Comment by NYFDmods** [comment]
Hey sorry my laptop charger died on me and so was my laptop.. I’m trying but I keep getting files not found

Sent from my iPhone

On Sep 16, 2017, at 4:34 AM, Angristan <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #93<https://github.com/Angristan/OpenVPN-install/issues/93>.
